File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieReview.java
package br.com.popularmovies.services.movieService.response;

import com.squareup.moshi.Json;

public class MovieReview {
    @Json(name = "author")
    private String author;

    @Json(name = "content")
    private String content;

    public String getAuthor() {
        return author;
    }

    public String getContent() {
        return content;
    }
}

File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieReview.kt
package br.com.popularmovies.services.movieService.response

import android.os.Parcelable
import com.squareup.moshi.Json
import kotlinx.android.parcel.Parcelize

@Parcelize
data class MovieReview(
    @field:Json(name = "author")
    val author: String = "",
    @field:Json(name = "content")
    val content: String = ""
) : Parcelable
